src/components/Header.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import { NavLink } from 'react-router-dom';
+
+export const Header = () => {
+  const createClassName = ({ isActive }: { isActive: boolean }) => {
+    const createdClass = 'nav-link';
+
+    if (isActive) {
+      return `${createdClass} active`;
+    }
+
+    return createdClass;
```

Comment:
can we use `classnames` lib for this logic?

Reply:
Yes, but for this way i need to get current path of url from props, so it is a litle bit harder